### PR TITLE
Fix where the events time is shown twice for the same timezone

### PR DESCRIPTION
**Pull request by thread-koder**

### CODE REVIEW

File: components/Events/TimeCard.vue [hunk 1]
```diff
@@ -56,6 +58,12 @@ export default {
       const currentDate = new Date().getTime()
       return currentDate < endDate
     },
+    isTheSameTimezone() {
+      const LocalTimezone = Intl.DateTimeFormat().resolvedOptions().timeZone
+      const ammanTimezone = 'Asia/Amman'
+
+      return LocalTimezone === ammanTimezone
```
Comment (mahasneh) [inline components/Events/TimeCard.vue:65]: Capital Z

File: components/Events/TimeCard.vue [hunk 2]
```diff
@@ -56,6 +58,12 @@ export default {
       const currentDate = new Date().getTime()
       return currentDate < endDate
     },
+    isTheSameTimezone() {
+      const LocalTimezone = Intl.DateTimeFormat().resolvedOptions().timeZone
```
Comment (mahasneh) [inline components/Events/TimeCard.vue:62]: small L

File: components/Events/TimeCard.vue (hunk 1 above)
Comment (thread-koder) [reply components/Events/TimeCard.vue]: Fixed

File: components/Events/TimeCard.vue [hunk 3]
```diff
@@ -56,6 +58,12 @@ export default {
       const currentDate = new Date().getTime()
       return currentDate < endDate
     },
+    isTheSameTimeZone() {
```
Comment (mahasneh) [inline components/Events/TimeCard.vue:61]: isLocalTimeZoneAmman()

**Pull request merged**